### ISSUE TITLE
Accessibility: aria-labels

### ISSUE THREAD

**Issue by gmrabian** [coforma/swift-tech-challenge#210]
**Describe the bug**
Some aria labels for landmarks are generic like "banner" and "navigation".
Many of these come from the `@trussworks/uswds` components.

** Current behavior **
Landmarks are generic and don't help with getting around the website.

**To Reproduce**
Steps to reproduce the behavior (on MacOS):
1. Go to the homepage
2. Start VoiceOver
3. Open the rotor and view "Landmarks"

**Expected behavior**
Labels should be human readable and clear on intent

**Screenshots**

## Definition of Done
[Definition of Done](https://github.com/coforma/swift-tech-challenge/blob/main/documentation/product/definition_of_done.md)
